components/Channel.php

```diff
@@ -81,6 +81,12 @@ public function defineProperties()
                 'type'        => 'dropdown',
                 'group'       => 'Links',
             ],
+            'css' => [
```

Comment:
if you can rename this property to `includeStyles` instead that would be better

Reply:
Corrected